lib/matplotlib/axes/_base.py

```diff
@@ -1286,16 +1286,16 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
 
     def get_adjustable(self):
         """
-        Returns the adjustable parameter, *{'box', 'datalim'}* that defines
-        which parameter the Axes will change to achieve a given aspect.
+        Return whether the Axes will adjust its physical dimension ('box') or
+        its data limits ('datalim') to achieve the desired aspect.
 
         See Also
         --------
         matplotlib.axes.Axes.set_adjustable
-            defining the parameter to adjust in order to meet the required
+            Set the parameter to adjust in order to achieve the required
```

Comment:
That sounds complicated
```suggestion
            Set how the Axes adjusts to the required aspect ratio.
```

Reply:
better, changed